utensor_cgen/template_env.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding:utf8 -*-
```

Comment:
What exactly does this file do, can you add comments

Reply:
Oh, I just figure out that it's better to manage the template files with `Environment` object in jinja2.
http://jinja.pocoo.org/docs/2.10/api/

With it, you have better control over templates rendering.

Reply:
👍